ckanext/geodatagov/logic.py

```diff
@@ -459,6 +459,9 @@ def translate_spatial(old_spatial):
                    '"coordinates": [[[{minx}, {miny}], [{minx}, {maxy}], '
                    '[{maxx}, {maxy}], [{maxx}, {miny}], [{minx}, {miny}]]]}}')
 
+    # replace all instances of '+' as this creates bad JSON https://github.com/GSA/data.gov/issues/3549
+    old_spatial = old_spatial.replace('+', '')
+
```

Comment:
I think we want to try to leave `old_spatial` as is, as that is the value given to us by the provider. I would rather see this change down near line 480, as the start of the transformation we do in that specific use case.

Reply:
I agree with @jbrown-xentity (having more context on this now)

Reply:
I had chained the fix into line 481 `old_spatial.strip().split(',')` but then I realized that old_spatial is getting loaded in at [L467](https://github.com/GSA/ckanext-geodatagov/blob/spatial-plus-sign-fix/ckanext/geodatagov/logic.py#L467) too and that we need to transform it before that.

Reply:
Is anyone clear where initial `spatial` is getting saved into `old_spatial`? If I can find that I can ensure the initial value gets saved unchanged.